styles/main.css

```diff
@@ -394,6 +394,11 @@ a.twitter-mention-button {
     position: relative;
 }
 
+.bannerbox__logo {
+    height: 50px;
+    width: 15%;
+}
+
```

Comment:
These images are currently skewed:
![image](https://cloud.githubusercontent.com/assets/1155816/20057315/c96e1570-a4e2-11e6-8304-0d18f5a1db38.png)

Can you please change this to something like the following, which should fix it:
```
.bannerbox__logo {
    max-height: 50px;
    width: auto;
    margin-right: 5px;
}
```